rollup.config.js

```
@@ -48,6 +48,11 @@ const getBabelOptions = ({useESModules}) => ({
 })
 
 const commonjsOptions = {
+  include: [
+    /\/node_modules\/react\//,
```

This here creates the issue with flow.

---

@TrySound @kof

---

I get it. We use ancient rollup version which silently swallowed commonjs errors without emitting files.

---

Upgrading it in separate branch

---

So upgrading to rollup v1 would solve the issue?

---

It will show the real error which is swallowed now.

---

Seems like the error was in missed `react-display-name` package which is a transitive dependency of `theming`.